aemedge/styles/styles.css

```diff
@@ -194,11 +194,21 @@ a:hover {
 }
 
 .button-container {
-  strong, em {
+  strong,
+  em {
     display: block;
   }
-  & +p {margin-top: 0;}
 
+  & + p {
+    margin-top: 0;
+  }
+
+  a.button {
```

Comment:
This should be the same as ` .button` below, right?

Reply:
right added hover effect as well from below

Reply:
> This should be the same as  .button below, right?

... so why do we need .button and a.button?

Reply:
for the very same reason why it was not populating initially . I.e 

<img width="1235" alt="image" src="https://github.com/urfuwo/hlx-test/assets/41055186/87b68802-ca00-49fe-a5b3-4fba99b9064a">

vs 

<img width="1235" alt="image" src="https://github.com/urfuwo/hlx-test/assets/41055186/69cea6cf-304e-42f3-accd-135a2b46d4be">

hence the anchor tags need styling of its own .

Reply:
Because the anchor tags are a direct child in this case . They need special styling .